homeassistant/components/workday/__init__.py

```diff
@@ -1,15 +1,28 @@
 """Sensor to indicate whether the current day is a workday."""
 from __future__ import annotations
 
+from holidays import list_supported_countries
+
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError
 
-from .const import PLATFORMS
+from .const import CONF_COUNTRY, CONF_PROVINCE, LOGGER, PLATFORMS
 
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Workday from a config entry."""
 
+    country: str = entry.options[CONF_COUNTRY]
+    province: str | None = entry.options.get(CONF_PROVINCE)
+    if country and country not in list_supported_countries():
+        LOGGER.error("There is no country %s", country)
```

Comment:
Don't log an error. There will already be an error logged by the config entries helper. Pass the message to the exception instead. The message will be included in the log.